src/app/workers/ship.ts

```diff
@@ -20,8 +20,8 @@ class Ship {
   getCopy(): Ship {
     const ret = Ship.Ships.pop() || new Ship();
     ret.id = this.id;
-    ret.armor.fromDecimal(this.armor);
```

Comment:
What is this trying to fix?

The whole point of using the instance is to reuse the instance instead of creating a new object.

`Decimal.fromMantissaExponent` is just a wrapper for new Decimal.
https://github.com/Patashu/break_infinity.js/blob/2465fc24ad69550f4bba82a5ddd47dd2f5cf9a46/src/break_infinity.ts#L126-L128

Eg: `ret.armor.fromMantissaExponent` would reuse the instance.

Reply:
It wasn't needed